README.md

```diff
@@ -1,24 +1,38 @@
-# README
+# entrylvl plataform
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is a blog dedicated to the entrylvl, feel free to use any of the code, but the logo and the images we reserve to rights.
```

Comment:
I think would it be better if instead of "we reserve to rights" we use "we reserve all rights"